README.md

```diff
@@ -14,7 +14,19 @@ Alternatively, you could clone this repository and run the app directly on your
 ## Usage
 Open the app for the first time and it will prompt you to configure the Adyen Test Cards autofill service. Once done, you can autofill payment method fields as in the video below:
 
-https://github.com/user-attachments/assets/df20d562-af1d-4703-88a8-43b87a86039b
+| Android                     | Web                 |
+|-----------------------------|---------------------|
+| ![Android][mystore.android] | ![Web][mystore.web] |
+
+> [!NOTE]  
+> Autofilling on Chrome does not work by default. To enable autofilling on Chrome follow these steps:
+> 1. Open Chrome and go to `chrome://flags#enable-autofill-virtual-view-structure` *
+> 2. Enable the flag and restart Chrome * 
+> 3. Open the settings in Chrome and navigate to "Autofill Services" 
+> 4. Select "Autofill using another service"
+> 5. Restart Chrome
+>
+> \* These steps are not needed on Chrome version 131 and later
```

Comment:
Since most people will have Chrome 131+ in a few weeks, the first 2 steps will useless for most. And we only mention that they are not needed under the last step. What if we directly say for "Chrome below 131..."? Or move these 2 steps to the bottom?